numba/core/errors.py

```diff
@@ -314,6 +328,13 @@ def termcolor():
             _termcolor_inst = HighlightColorScheme(scheme)
         return _termcolor_inst
 
+
+pedantic_warning_info = """
+This warning came from an internal pedantic check. Please report the warning
+message and traceback, along with a minimal reproducer at:
+https://github.com/numba/numba/issues/new?template=bug_report.md
+"""
+
 feedback_details = """
 Please report the error message and traceback, along with a minimal reproducer
 at: https://github.com/numba/numba/issues/new
```

Comment:
Do you want to update this URL too, while there?

Reply:
Let's do this in a future PR.

Reply:
i've made a task issue for it: https://github.com/numba/numba/issues/7261